src/fsharp/FSharp.Core.Unittests/FSharp.Core/Microsoft.FSharp.Collections/ArrayModule.fs

```diff
@@ -148,15 +148,15 @@ type ArrayModule() =
     member this.ChunkBySize() =
 
         // int Seq
-        Assert.AreEqual([| [|1..4|]; [|5..8|] |], Array.chunkBySize 4 [|1..8|])
-        Assert.AreEqual([| [|1..4|]; [|5..8|]; [|9..10|] |], Array.chunkBySize 4 [|1..10|])
-        Assert.AreEqual([| [|1|]; [|2|]; [|3|]; [|4|] |], Array.chunkBySize 1 [|1..4|])
+        Assert.IsTrue([| [|1..4|]; [|5..8|] |] = Array.chunkBySize 4 [|1..8|])
```

Comment:
It's not perfect, no. Failures will still have a stack pointing to the line that failed, thoughm so it's not too bad.  And FWIW this is the pattern used for most of the tests already (due to the same reason, that older NUnit has broken `AreEqual` for complicated collections).